client/src/components/Components.tsx

```diff
@@ -3,11 +3,13 @@ import { Input } from "@material-ui/core";
 import { EMPTY_PLACEHOLDER } from "../constants";
 
 export function TextInputField({
+  elementId,
```

Comment:
nit: Just id should be good to stay conformed with HTML convention
```suggestion
  id,
```

Reply:
changed